### ISSUE TITLE
JSON to Spreadsheet - Improve array support (when not using a schema)

### ISSUE THREAD

**Issue by Bjwebb** [OpenDataServices/flatten-tool#32]
```
E         -                'tag': ['tender'],
E         ?                       -        -
E         +                'tag': 'tender',
```

(Diff taken from new roundtrip test:
https://github.com/open-contracting/flattening-ocds/commit/523c837a07d303c2463300d2807d227a92cef4e6)

**Comment by jpmckinney** [comment]
Related to #26, which also depends on how to best represent simple arrays.

The current behavior is to put semi-colons between values, but if there is only one value, then the array is lost in the round-trip. The simplest solution might be to add a final semi-colon, but this creates a new edge case, if the last value in an array was null or empty in the original data.

I think the preferred solution is to not use semi-colons as the default behavior, but to split array values into e.g. tender/0, tender/1. Users should be able to opt-in to the use of semi-colons, but at the cost of having to either supply a schema or living with errors on edge cases.

In other words, the default behavior should be the most robust, and users should have to opt-in to error-prone alternatives.
